federationapi/routing/keys.go

```diff
@@ -217,10 +221,7 @@ func NotaryKeys(
 		}
 	}
 
-	var response struct {
-		ServerKeys []json.RawMessage `json:"server_keys"`
-	}
-	response.ServerKeys = []json.RawMessage{}
```

Comment:
Bad refactor. You forgot to do this, so no keys will be `server_keys: null` not `server_keys: []`.